recipes/rapidcheck/all/conanfile.py

```diff
@@ -114,7 +110,7 @@ def package(self):
         )
 
     @staticmethod
```

Comment:
I think this might no longer need to be a `staticmethod` if you're passing self as the first argument :)